services/sleeper/docker-compose-meta.yml

```diff
@@ -5,23 +5,27 @@ services:
         io.simcore.authors: '{"authors": [{"name": "Manuel Guidon", "email": "guidon@itis.swiss",
           "affiliation": "IT''IS Foundation"}, {"name": "Odei Maiz", "email": "maiz@itis.swiss",
           "affiliation": "IT''IS Foundation"}, {"name": "Andrei Neagu", "email": "neagu@itis.swiss",
-          "affiliation": "IT''IS Foundation"}]}'
+          "affiliation": "IT''IS Foundation"}, {"name": "Sylvain Anderegg", "email":
+          "anderegg@itis.swiss", "affiliation": "IT''IS Foundation"}]}'
         io.simcore.contact: '{"contact": "maiz@itis.swiss"}'
         io.simcore.description: '{"description": "A service which awaits for time
           to pass, two times."}'
         io.simcore.inputs: '{"inputs": {"input_1": {"displayOrder": 1, "label": "File
           with int number", "description": "Pick a file containing only one integer",
           "type": "data:text/plain", "fileToKeyMap": {"single_number.txt": "input_1"}},
           "input_2": {"displayOrder": 2, "label": "Sleep interval", "description":
-          "Choose an amount of time to sleep in range [0-65]", "defaultValue": 2,
+          "Choose an amount of time to sleep in range [0-900]", "defaultValue": 2,
```

Comment:
Q: Why are giving an upper limit? I guess we might test at some point even running the job for 3 hours?

Reply:
not sure who started with that upper limit... I can make it go to a day or 2

Reply:
actually removing the max